### PR TITLE
Log cleanup

### PR DESCRIPTION
**Pull request by skyshaha1**
Summary: Log Cleanup
Type: Bug Fix
Test Plan: Unit Tests & FullStack
Jira: RIALTO-297

### CODE REVIEW

**Review by skywojciechowskim — APPROVED** [review]

File: media/server/main/source/MediaPipelineServerInternal.cpp [hunk 1]
```diff
@@ -595,22 +617,14 @@ bool MediaPipelineServerInternal::haveDataInternal(MediaSourceStatus status, uin
         // Incrementing the counter allows us to track the occurrences where the status is other than OK or EOS. By
         // limiting the logging of the warning message to the number of NO_AVAILABLE_SAMPLES in a row, we prevent the
```
Comment (skywilliamsonl) [inline media/server/main/source/MediaPipelineServerInternal.cpp:618]: Update comment

File: media/server/main/source/MediaPipelineServerInternal.cpp [hunk 2]
```diff
@@ -531,14 +530,37 @@ bool MediaPipelineServerInternal::haveDataInternal(MediaSourceStatus status, uin
         return true;
     }
 
+    unsigned int &counter = m_noAvailableSamplesCounter[mediaSourceType];
     if (status != MediaSourceStatus::OK && status != MediaSourceStatus::EOS)
     {
-        RIALTO_SERVER_LOG_WARN("Data request for needDataRequestId: %u received with wrong status: %s",
-                               needDataRequestId, toString(status));
+        // Incrementing the counter allows us to track the occurrences where the status is other than OK or EOS. By
+        // limiting the logging of the warning message to the number of NO_AVAILABLE_SAMPLES in a row, we prevent the
```
Comment (skywilliamsonl) [inline media/server/main/source/MediaPipelineServerInternal.cpp:537]: Update comment

File: media/server/main/source/MediaPipelineServerInternal.cpp [hunk 3]
```diff
@@ -531,14 +530,37 @@ bool MediaPipelineServerInternal::haveDataInternal(MediaSourceStatus status, uin
...
+        // warning from being repeatedly logged.
+
+        counter++;
```
Comment (skywojciechowskim) [inline media/server/main/source/MediaPipelineServerInternal.cpp:540]: ++counter;

File: media/server/main/source/MediaPipelineServerInternal.cpp [hunk 4]
```diff
@@ -595,22 +617,14 @@ bool MediaPipelineServerInternal::haveDataInternal(MediaSourceStatus status, uin
         // Incrementing the counter allows us to track the occurrences where the status is other than OK or EOS. By
         // limiting the logging of the warning message to the number of NO_AVAILABLE_SAMPLES in a row, we prevent the
         // warning from being repeatedly logged.
+
+        counter++;
```
Comment (skywojciechowskim) [inline media/server/main/source/MediaPipelineServerInternal.cpp:621]: ++counter

**Review by skywilliamsonl — APPROVED** [review]

**Review by skywojciechowskim — APPROVED** [review]

**Pull request merged**